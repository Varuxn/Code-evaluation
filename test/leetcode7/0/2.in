9
873 550 16 640 169 733 560 196 873
