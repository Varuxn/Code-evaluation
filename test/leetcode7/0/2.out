323 534 16 471 169 173 364 196 873 